dashboard/app/assets/javascripts/utils.js.erb

```diff
@@ -9,18 +9,6 @@ var addClickTouchEvent = function(element, handler) {
   });
 };
 
-function onURLBlocked(base_url, callback) {
-  $('<img width=1 height=1>')
-      .on("error", callback)
-      .attr("src", base_url + "?" + Math.random())
```

Comment:
I've specifically avoided supporting that scenario in the HTTPS redirection setup, because falling back to HTTPS to do the detection logic on a regular basis would eliminate any of the security guarantees HTTPS provides in the first place. If we want to continue supporting the same scenario for the Youtube detection, I'll add the random query string to the URL in the YouTube detection logic. That way we can still reuse the same code for both